e2etest/e2etest_suite_test.go

```diff
@@ -34,11 +34,17 @@ import (
 	e2econfig "k8s.io/kubernetes/test/e2e/framework/config"
 )
 
+// use ephemeral port for pod, instead of well-known port (tcp/80)
+var servicePodPort uint
+var skipDockerCmd bool
+
 // handleFlags sets up all flags and parses the command line.
 func handleFlags() {
 	e2econfig.CopyFlags(e2econfig.Flags, flag.CommandLine)
 	framework.RegisterCommonFlags(flag.CommandLine)
 	framework.RegisterClusterFlags(flag.CommandLine)
+	flag.UintVar(&servicePodPort, "service-pod-port", 80, "port number that pod opens, default: 80")
```

Comment:
Did you consider just changing the port numbers in all cases to something that would work for everyone instead of making it configurable?

Reply:
The security policy depends on each environments, platforms. Some platform might accept non well-known port and other platform might accept only more than tcp/3000. I don't know perfect port number which accepts everywhere, hence I make it configurable.

Please let me know if you know the perfect port number which not be blocked by any firewall (with any firewall configurations), cloud platform and other aspects. I'm happy to change it immediately.

Reply:
In addition, Kubernetes uses port 80, as default.

Reply:
OK, not a big deal.  I know we really just have 2 systems we're dealing with at the moment, so I'd be fine with a simple change that worked for both CI systems.  What you have is fine too though.